setup.py

```diff
@@ -53,14 +53,14 @@ def has_libc_iconv():
     libraries=LIBS
 )
 
-setup(name='zbar',
-        version='1.1',
+setup(name='zbar-py',
+        version='1.2rc1',
         description='zbar package',
         url='https://github.com/rounaksingh/zbar-py',
         author='Zachary Pincus, Rounak Singh',
         author_email='zpincus@gmail.com, rounaksingh17@gmail.com',
         ext_modules=[zbar],
         packages=['zbar'],
-        install_requires=['numpy'],
+        install_requires=['numpy','scipy'],
```

Comment:
Scipy is a very heavyweight requirement, especially since it's just for image reading -- which is not even a core part of the zbar functionality. There are **lots** of ways to get images into numpy arrays; using the most appropriate for any specific application should be the zbar user's responsibility. By all means have the scipy-based image reading as example code, though!